edx_sga/sga.py

```diff
@@ -561,9 +561,21 @@ def enter_grade(self, request, suffix=''):
         Persist a score for a student given by staff.
         """
         require(self.is_course_staff())
+        score = request.params.get('grade', None)
         module = StudentModule.objects.get(pk=request.params['module_id'])
+        if not score:
+            log.error(
+                "[enter_grade] Please enter valid grade for course:%s module:%s student:%s",
```

Comment:
This is an odd error message for a log. Wouldn't something like `enter_grade: invalid grade submitted for course:%s module:%s student:%s` make more sense?

The square brackets aren't used anywhere else either.